tests/testImageDecorrelation.py

```diff
@@ -245,19 +265,19 @@ def _testImages(self):
 
         # Expected (ideal) variance of difference image
         expected_var = self.svar + self.tvar
-        print('Expected variance:', expected_var)
+        print('EXPECTED VARIANCE:', expected_var)
```

Comment:
Could you please say why you think the print statement is useful here? It would get gobbled up during jenkins runs, and not normally seen by a use, unless they have reason to look at the test output (e.g., the test failed). If there's some information not being captured by the current asserts, maybe we need more asserts?